war/src/main/webapp/help/system-config/master-slave/usage.html

```diff
@@ -19,11 +20,11 @@
       that project is restricted to certain nodes using a label expression,
       and that expression matches this node&apos;s name and/or labels.
 
-      This allows a slave to be reserved for certain kinds of jobs.
+      This allows a node to be reserved for certain kinds of jobs.
```

Comment:
It applies to both master and agents (formerly slaves). Master is not an agent, but a node.